src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementService.java

```diff
@@ -372,11 +375,33 @@ public void cancelTask(String taskId,
         if (isRelease2EndpointsFeatureEnabled) {
             //Lock & update Task
             TaskResource task = findByIdAndObtainLock(taskId);
+            CFTTaskState previousTaskState = task.getState();
             task.setState(CFTTaskState.CANCELLED);
-            //Perform Camunda updates
-            camundaService.cancelTask(taskId);
-            //Commit transaction
-            cftTaskDatabaseService.saveTask(task);
+
+            String camundaCftTaskState = camundaService.getCftTaskState(taskId);
```

Comment:
The naming is misleading here.  This is the Camunda History table cftTaskState variable value.

Reply:
I don't see the change here?